src/util/map.test.js

```diff
@@ -295,6 +295,7 @@ describe("util/map", () => {
       // $ExpectFlowError
       const _unused_badMap: Map<string, number> = MapUtil.merge([
         numberMap,
+        // $ExpectFlowError
         stringMap,
       ]);
```

Comment:
In this case we need two comments because this array is an invalid argument as all it's values types are expected to be the same. And the variable assignment is invalid as well, because it's type does not match the return type.